<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SensorWidget</class>
 <widget class="QWidget" name="SensorWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>849</width>
    <height>491</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <property name="topMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QSpinBox" name="spinHardwareId">
       <property name="maximum">
        <number>999999</number>
       </property>
       <property name="value">
        <number>20000</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox">
       <item>
        <property name="text">
         <string>Scalar</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Array</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbSensorType">
       <property name="text">
        <string>Set sensor</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLineEdit" name="linehost">
       <property name="text">
        <string>192.168.2.10</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinPort">
       <property name="maximum">
        <number>65535</number>
       </property>
       <property name="value">
        <number>7778</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pb_setClient">
       <property name="text">
        <string>Set client</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Array Size</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinDataSize">
       <property name="maximum">
        <number>9999</number>
       </property>
       <property name="value">
        <number>2400</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbSetLength">
       <property name="text">
        <string>Set Length</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Critical upper</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinCUp">
       <property name="maximum">
        <number>16535</number>
       </property>
       <property name="value">
        <number>900</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Warning upper</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinWUp">
       <property name="maximum">
        <number>16535</number>
       </property>
       <property name="value">
        <number>800</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Warning lower</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinWDown">
       <property name="maximum">
        <number>16535</number>
       </property>
       <property name="value">
        <number>200</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Critical lower</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinCDown">
       <property name="maximum">
        <number>16535</number>
       </property>
       <property name="value">
        <number>100</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbSetThreshold">
       <property name="text">
        <string>Set threshold</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QSpinBox" name="spinRandomInterval">
       <property name="maximum">
        <number>999</number>
       </property>
       <property name="value">
        <number>5</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pb">
       <property name="text">
        <string>Set random interval Â±</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QSpinBox" name="spinMS">
       <property name="maximum">
        <number>100000</number>
       </property>
       <property name="singleStep">
        <number>1</number>
       </property>
       <property name="value">
        <number>1000</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbInterval">
       <property name="text">
        <string>Set interval</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QComboBox" name="comboBox_2">
       <item>
        <property name="text">
         <string>0</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>4</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>8</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>16</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>4096</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Board Status</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="chartLayout">
     <property name="spacing">
      <number>0</number>
     </property>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="scalarDataLayout">
     <item>
      <widget class="QSpinBox" name="scalarDataSpinBox">
       <property name="maximum">
        <number>65535</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="scalarDataButton">
       <property name="text">
        <string>Scalar Data</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="pbStartStop">
       <property name="text">
        <string>Start</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
